components/alternate-accessions.js

```diff
@@ -0,0 +1,26 @@
+// node_modules
+import PropTypes from "prop-types";
+
+/**
+ * // Display the alternate accessions, normally below the header line in objects.
+ */
+export default function AlternateAccession({ alternateAccessions = [] }) {
+  if (alternateAccessions?.length > 0) {
+    return (
+      <h4 className="text-gray-500">
+        {alternateAccessions.length === 1 ? (
+          <span>Alternate Accession: {alternateAccessions[0]}</span>
+        ) : (
+          <span>Alternate Accessions: {alternateAccessions.join(", ")}</span>
```

Comment:
Change both of these from `<span>…</span>` to `<>…</>` to save a bit of HTML. It’ll have the same function but won’t generate the non-necessary spans.